### ISSUE TITLE
Graphback cli is not installing globally

### ISSUE THREAD

**Issue by wtrocki** [aerogear/graphback#654]
We got numerous repots on issues with our CLI. Needs more investigation

**Comment by wtrocki** [comment]
```
npx create-node-app test
cd test 
graphback config


{ Error: ENOENT: no such file or directory, open '/Users/wtrocki/Projects/sandbox/amplify-datastore/test//model/Note.graphql'
```

**Comment by wtrocki** [comment]
FYI @kingsleyzissou

**Comment by wtrocki** [comment]
Resolved and released in graphback 0.10.4

**Comment by wtrocki** [comment]
We had undocumented requirement to have model folder inside the project. 
Now we creating this folder everytime
